### PR TITLE
Aj docs update

### PR DESCRIPTION
**Pull request by ajrice6713**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

## Brief Summary of changes
Added documentation that covers setting a Final Destination/Failover URI

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-AJ-Docs-Update.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-AJ-Docs-Update.s3-website-us-east-1.amazonaws.com/

File: numbers/guides/managingLineFeatures.md [hunk 1]
```diff
@@ -114,5 +115,59 @@ In addition to the name and address information inherent in a DLDA update, the D
...
+To set the Final Destination URI to a SIP address, use the format sip:{address}@{host}
+
+To set the Final Destination URI to a PSTN address, use the format +13332221111@PSTN
+
+More detailed informaion on setting a Final Destination URI can be found [here](https://www.bandwidth.com/blog/establishing-automatic-failover-for-routing-calls-in-case-of-failure-final-destination-uri/).
+
+### How to Establish Automatic Failover for a Location (Sip Peer)
+To enable a Final Destination URI on a location, you can make a `PUT` request to `accounts/{accountId}/sites/{siteId}/sippeers/{sippeerId}`. 
+
+```
```
Comment (dtolb) [inline numbers/guides/managingLineFeatures.md]: Include ` ```http` to get nice formatting

```http
PUT https://dashboard.bandwidth.com/api/account/{accountId}/sites/{siteId}/sippeers/{sippeerId}
Authorization: Basic {base64_encoded_credentials)
Content-Type: application/xml
<SipPeer>
 <PeerName>{location_name}</PeerName>
 <FinalDestinationUri>{final_destination}</FinalDestinationUri>
 <IsDefaultPeer>true</IsDefaultPeer>
</SipPeer>
```

File: numbers/guides/managingLineFeatures.md [hunk 2]
```diff
@@ -114,5 +115,59 @@ In addition to the name and address information inherent in a DLDA update, the D
...
+ <FinalDestinationUri>{final_destination}</FinalDestinationUri>
+ <IsDefaultPeer>true</IsDefaultPeer>
+</SipPeer>
+
+```
+
+### How to Establish Automatic Failover for a Phone Number 
+To enable a Final Destination URI on an individual number, you can create a `TnOptions` order to update the line features. Other `TnOptionGroup` features can be included when adding a Final Destination URI. You may update up to 5,000 numbers at a time. 
+
+```
```
Comment (dtolb) [inline numbers/guides/managingLineFeatures.md]: Same as above

**Review by dtolb — CHANGES_REQUESTED** [review]
add ` ```http ` to code samples for syntax highlighting

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-AJ-Docs-Update.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-AJ-Docs-Update.s3-website-us-east-1.amazonaws.com/

**Pull request merged**